dubbo-dependencies/dubbo-dependencies-zookeeper/pom.xml

```diff
@@ -28,15 +28,20 @@
 
     <groupId>org.apache.dubbo</groupId>
     <artifactId>dubbo-dependencies-zookeeper</artifactId>
-    <version>2.7.2-SNAPSHOT</version>
+    <version>${revision}</version>
     <packaging>pom</packaging>
 
+    <properties>
+        <revision>2.7.2-SNAPSHOT</revision>
```

Comment:
Can we remove this later?

Reply:
I think we can try later, but now, we can't.

`dubbo-dependencies/dubbo-dependencies-zookeeper/pom.xml`'s parent pom is not set to `dubbo-parent`, only when its parent is `dubbo-parent` can we do that.

Reply:
I understand, it sound minor to me. It can be fixed later.